### PR TITLE
Added another parameter to support localtime based timestamps

### PR DESCRIPTION
**Pull request by dzorrilla**

### CODE REVIEW

File: client/common/directives/DateTimePicker/DateTimePicker.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ angular.module('dashboard.directive.DateTimePicker', [
         elem.datetimepicker({
           format: scope.format,
           useCurrent: false,
-          defaultDate: scope.defaultDate ? moment(scope.defaultDate, scope.format) : undefined,
+          defaultDate: scope.defaultDate ? moment(scope.defaultDate, scope.format) : (scope.$parent.data.start ? scope.$parent.data.start : undefined),
```
Comment (coolwr) [inline client/common/directives/DateTimePicker/DateTimePicker.js]: What is `$parent.data.start`? Is this custom for the project you are working on? If so, can you use the `defaultDate` property?

**Comment by dzorrilla** [comment]
@coolwr updated and ready for review

File: client/common/directives/ModelField/ModelField.js [hunk 2]
```diff
@@ -316,7 +316,18 @@ angular.module('dashboard.directives.ModelField', [
...
-        
+
+        if (property.display.convertToLocalTime === false) {
+          // remove the 'Z' from the end of the timestamp so that it is not converted to local time
+          if (scope.data[scope.key]) {
+            scope.data[scope.key] = scope.data[scope.key].substring(0, scope.data[scope.key].length-1);
+          }
+        }
+
+        if (property.display.defaultValueUsingModelKey) {
```
Comment (coolwr) [inline client/common/directives/ModelField/ModelField.js]: Can you do some testing around this? looks like you are forcing the default value even if the scope.data[scope.key] has a value already.

File: client/common/directives/ModelField/ModelField.js [hunk 3]
```diff
@@ -316,7 +316,18 @@ angular.module('dashboard.directives.ModelField', [
           if (!property.display.options) property.display.options = {};
           if (!property.display.options.format) property.display.options.format = "YYYY-MM-DD  h:mm A";
         }
-        
+
+        if (property.display.convertToLocalTime === false) {
```
Comment (coolwr) [inline client/common/directives/ModelField/ModelField.js]: Can you add you combine 320 and 322 to do `if (scope.data[scope.key] && property.display.convertToLocalTime === false)`

**Pull request merged**